### ISSUE TITLE
Improve the README

### ISSUE THREAD

**Issue by LouisCAD** [LouisCAD/Splitties#58]
Improvement path:
- Add snippets to compare with and without, as done in Android KTX README
- The list of the splits can be made thinner by replacing headings by bullet points
- The related splits may be grouped (views group, viewdsl group, others group…)

Any other ideas to improve it further are welcome!

**Comment by LouisCAD** [comment]
Now that the README is more concise, it's time to add some snippets that show a selection of the benefits of using Splitties.

**Comment by AdamMc331** [comment]
I think the READMe is a lot better! If you're going to add samples to the readme, I would try to keep them limited so that it doesn't get too long again.

Consider just using samples from what you think are the most popular splits - my guesses are things like appctx, snackbar, resources, fragmentargs as some examples. A developer might not be immediately interested in stetho or some of the selectable views, in my experience - but samples for that in the individual modules is good.

TL;DR I think this is on the right track! Just be careful not to go overboard in the main README again.

**Comment by LouisCAD** [comment]
@AdamMc331 I could also collapse by default the comparison for modules like selectableviews and stetho-init like I did for the outline in the previous version of the README. WDYT?
